<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>311</width>
    <height>355</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#Widget{
background-color: rgb(85, 255, 255);
}
QScrollBar:vertical
{
    width:8px;
    background:rgba(0,0,0,0%);
    margin:0px,0px,0px,0px;
    padding-top:9px;  
    padding-bottom:9px;
}
QScrollBar::handle:vertical
{
    width:8px;
    background:rgba(0,0,0,25%);
    border-radius:4px;   
    min-height:20;
}
QScrollBar::handle:vertical:hover
{
    width:8px;
    background:rgba(0,0,0,50%);  
    border-radius:4px;
    min-height:20;
}
QScrollBar::add-line:vertical  
{
    height:9px;width:8px;
    border-image:url(:/images/a/3.png);
    subcontrol-position:bottom;
}
QScrollBar::sub-line:vertical  
{
    height:9px;width:8px;
    border-image:url(:/images/a/1.png);
    subcontrol-position:top;
}
QScrollBar::add-line:vertical:hover   
{
    height:9px;width:8px;
    border-image:url(:/images/a/4.png);
    subcontrol-position:bottom;
}
QScrollBar::sub-line:vertical:hover 
{
    height:9px;width:8px;
    border-image:url(:/images/a/2.png);
    subcontrol-position:top;
}
QScrollBar::add-page:vertical,QScrollBar::sub-page:vertical   
{
    background:rgba(0,0,0,10%);
    border-radius:4px;
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QWidget" name="myWidget" native="true">
     <property name="styleSheet">
      <string notr="true">*{
outline:0px;
}
#myWidget{
background-color: rgb(255, 255, 255);
}
#widget_left{
background-color: rgb(248, 249, 250);
}
#label_title{
font: 11pt &quot;微软雅黑&quot;;
}
#label_txl{
font: 14pt &quot;微软雅黑&quot;;
font-weight:bold;
}
#pushButton_close,#pushButton_min{
font: 15pt &quot;微软雅黑&quot;;
border:none;
}
#pushButton_list,#pushButton_set{
border:none;
font: 9pt &quot;微软雅黑&quot;;
}
#pushButton_close:hover{
background-color: rgb(255, 24, 24);
color:white;
}
#pushButton_close:pressed{
background-color: rgb(255, 0, 0);
color:white;
}
#pushButton_min:hover{
background-color: rgb(239, 245, 248);
}
#pushButton_min:pressed{
background-color: rgb(53, 174, 255);
}
#lineEdit{
border-radius:7px;
font: 10pt &quot;微软雅黑&quot;;
background-color: rgb(248, 249, 250);
}
#lineEdit_name,#lineEdit_number,#lineEdit_path{
border:1px solid rgba(200,200,200,0.3);
border-radius:10px;
font: 10pt &quot;微软雅黑&quot;;
background-color: white;
}
#lineEdit_name:hover,#lineEdit_number:hover,#lineEdit_path:hover{
border:1px solid rgb(0, 170, 255);
}
#lineEdit_name:focus,#lineEdit_number:focus,#lineEdit_path:focus{
border:1px solid rgb(0, 170, 255);
}
#pushButton_commit,#pushButton_file{
font: 10pt &quot;微软雅黑&quot;;
color:white;
border-radius:10px;
border:none;
background-color: rgb(100, 219, 100);
}
#pushButton_commit:hover,#pushButton_file:hover{
background-color: rgb(72, 162, 72);
}
#lineEdit:focus{
background-color: rgb(244, 244, 244);
}
QListWidget{
	outline:0px;
	border:none;
}
QListWidget::item{
	border:none;
	height:20px;
	border-radius:10px;
	color:black;
}
QListWidget::item:hover{
	background-color: rgb(228, 228, 228);
}
QListWidget::item:selected{
	background-color: rgb(228, 228, 228);
}

#pushButton_delete,#pushButton_add,#pushButton_change{
	border-radius:9px;
	background-color: rgb(244, 243, 243);
	font: 15pt ;
}
#pushButton_delete:hover{
	background-color: rgb(255, 61, 61);
	color:white;
}
#pushButton_change:hover{
	background-color: rgb(0, 170, 255);
	color:white;
}
#pushButton_add:hover{
	background-color: rgb(100, 219, 100);
	color:white;
}
#page_2 *{
font: 10pt &quot;微软雅黑&quot;;
}



</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="widget_left" native="true">
        <property name="minimumSize">
         <size>
          <width>70</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>16777215</height>
         </size>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>5</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_txl">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>通讯录</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_list" native="true">
           <property name="minimumSize">
            <size>
             <width>70</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>70</width>
             <height>16777215</height>
            </size>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>3</number>
            </property>
            <property name="topMargin">
             <number>5</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>5</number>
            </property>
            <item>
             <widget class="QLabel" name="label_list">
              <property name="minimumSize">
               <size>
                <width>14</width>
                <height>14</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>12</width>
                <height>12</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_list">
              <property name="text">
               <string>列表</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_set" native="true">
           <property name="minimumSize">
            <size>
             <width>70</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>70</width>
             <height>16777215</height>
            </size>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>3</number>
            </property>
            <property name="topMargin">
             <number>5</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>5</number>
            </property>
            <item>
             <widget class="QLabel" name="label_set">
              <property name="minimumSize">
               <size>
                <width>14</width>
                <height>14</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>12</width>
                <height>12</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_set">
              <property name="text">
               <string>设置</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>261</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_right" native="true">
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="leftMargin">
          <number>4</number>
         </property>
         <property name="topMargin">
          <number>4</number>
         </property>
         <property name="rightMargin">
          <number>4</number>
         </property>
         <property name="bottomMargin">
          <number>4</number>
         </property>
         <item>
          <widget class="QWidget" name="widget" native="true">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>20</height>
            </size>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label_title">
              <property name="text">
               <string>通讯录列表</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_min">
              <property name="minimumSize">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
              <property name="text">
               <string>-</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_close">
              <property name="minimumSize">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
              <property name="text">
               <string>×</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QStackedWidget" name="stackedWidget">
           <property name="focusPolicy">
            <enum>Qt::StrongFocus</enum>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Plain</enum>
           </property>
           <property name="currentIndex">
            <number>0</number>
           </property>
           <widget class="QWidget" name="page">
            <layout class="QGridLayout" name="gridLayout">
             <property name="bottomMargin">
              <number>3</number>
             </property>
             <item row="2" column="0" colspan="2">
              <widget class="QWidget" name="widget_2" native="true">
               <layout class="QHBoxLayout" name="horizontalLayout_6">
                <property name="spacing">
                 <number>0</number>
                </property>
                <property name="leftMargin">
                 <number>1</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>5</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QPushButton" name="pushButton_delete">
                  <property name="minimumSize">
                   <size>
                    <width>20</width>
                    <height>20</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>20</width>
                    <height>20</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>-</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QPushButton" name="pushButton_change">
                  <property name="minimumSize">
                   <size>
                    <width>20</width>
                    <height>20</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>20</width>
                    <height>20</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>✎</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLineEdit" name="lineEdit">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>17</height>
                </size>
               </property>
               <property name="inputMask">
                <string/>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
               <property name="placeholderText">
                <string>搜索</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QPushButton" name="pushButton_add">
               <property name="minimumSize">
                <size>
                 <width>18</width>
                 <height>18</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>18</width>
                 <height>18</height>
                </size>
               </property>
               <property name="text">
                <string> + </string>
               </property>
              </widget>
             </item>
             <item row="1" column="0" colspan="2">
              <widget class="QListWidget" name="listWidget"/>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="page_2">
            <layout class="QGridLayout" name="gridLayout_3">
             <item row="3" column="1">
              <widget class="QLabel" name="label">
               <property name="text">
                <string>排序方式</string>
               </property>
              </widget>
             </item>
             <item row="5" column="3">
              <spacer name="verticalSpacer_5">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="0" column="3">
              <spacer name="verticalSpacer_4">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::Maximum</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="1" column="1">
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>文件路径</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1" colspan="2">
              <widget class="QLineEdit" name="lineEdit_path"/>
             </item>
             <item row="2" column="3">
              <widget class="QPushButton" name="pushButton_file">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>16</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>50</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>选择</string>
               </property>
              </widget>
             </item>
             <item row="4" column="1">
              <widget class="QRadioButton" name="radioButton_name">
               <property name="text">
                <string>联系人名字</string>
               </property>
              </widget>
             </item>
             <item row="4" column="2">
              <widget class="QRadioButton" name="radioButton_number">
               <property name="text">
                <string>电话号码</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="page_3">
            <layout class="QGridLayout" name="gridLayout_2">
             <property name="verticalSpacing">
              <number>8</number>
             </property>
             <item row="2" column="0">
              <spacer name="horizontalSpacer_3">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="5" column="1">
              <spacer name="verticalSpacer_2">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="0" column="1">
              <spacer name="verticalSpacer_3">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::Maximum</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>60</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="1" column="2">
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="2" column="1">
              <widget class="QLineEdit" name="lineEdit_number">
               <property name="minimumSize">
                <size>
                 <width>120</width>
                 <height>23</height>
                </size>
               </property>
               <property name="focusPolicy">
                <enum>Qt::ClickFocus</enum>
               </property>
               <property name="contextMenuPolicy">
                <enum>Qt::DefaultContextMenu</enum>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
               <property name="dragEnabled">
                <bool>false</bool>
               </property>
               <property name="placeholderText">
                <string>联系人号码</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLineEdit" name="lineEdit_name">
               <property name="minimumSize">
                <size>
                 <width>120</width>
                 <height>23</height>
                </size>
               </property>
               <property name="focusPolicy">
                <enum>Qt::ClickFocus</enum>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
               <property name="placeholderText">
                <string>联系人姓名</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QPushButton" name="pushButton_commit">
               <property name="minimumSize">
                <size>
                 <width>30</width>
                 <height>24</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>9999</width>
                 <height>24</height>
                </size>
               </property>
               <property name="layoutDirection">
                <enum>Qt::LeftToRight</enum>
               </property>
               <property name="text">
                <string>添加</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
